graphbolt/include/graphbolt/csc_sampling_graph.h

```diff
@@ -115,6 +115,18 @@ class CSCSamplingGraph : public torch::CustomClassHolder {
   c10::intrusive_ptr<SampledSubgraph> InSubgraph(
       const torch::Tensor& nodes) const;
 
+  /**
+   * @brief Sample neighboring edges of the given nodes and return the induced
+   * subgraph.
+   *
+   * @param nodes The nodes from which to sample neighbors.
+   *
+   * @return An intrusive pointer to a SampledSubgraph object containing the
+   * sampled graph's information.
+   */
+  c10::intrusive_ptr<SampledSubgraph> SampleNeighbors(
+      const torch::Tensor& nodes) const;
```

Comment:
This is just for PLACE_HOLDER as parameters like `fanout` is missing?

Reply:
Yes